files/emr-3-qa/docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: "3.7"
+
+services:
+  # Proxy
+  proxy:
+    image: openmrs/openmrs-reference-application-3-gateway:latest
+    ports:
+      - 8084:80
```

Comment:
I _think_ you are forced to use a healthcheck. But I might be wrong.

Reply:
Alright. That should be trivial to add.